components/layout_2020/display_list/mod.rs

```diff
@@ -330,6 +330,16 @@ impl<'a> BuilderForBoxFragment<'a> {
     }
 
     fn build(&mut self, builder: &mut DisplayListBuilder) {
+        // If this box fragment has a transform applied that makes it take up no space,
```

Comment:
I think you can omit this check here, since you will not be adding this fragment to the `StackingContextTree`.